### DISCUSSION TITLE
Can't open a diagram

### DISCUSSION THREAD

**Discussion by speller** [jgraph/drawio#2391] [General]
I can't open a previously saved diagram from Google Drive. It waits for some time, then shows the Timeout error and next Access denied error. At the same time, I can create and open new diagrams. Could anyone help me to solve this issue?

**Comment by davidjgraph** [comment]
In drive.google.com there's a revision history of files. Try downloading the previous revision and drop that into the editor to see if it loads.

**Comment by speller** [comment]
The issue was the mobile internet connection. At home, I've opened it successfully. Using my 4G mobile internet, I was unable to open it. While no issues having video calls. I think this is a bug.
